### PR TITLE
improve example test by sorting output

### PR DESCRIPTION
**Pull request by ketsiambaku**
**What changed?**
Improving the example test by sorting the activities returned by Stats()
The example implementation range over a map and return the result in a slice. Because of this, the assertion against expected result would not always pass since the order is random. This made the test flaky

**Why?**

**How did you test it?**
unit tests

**Potential risks**

### CODE REVIEW

File: internal/common/debug/example_test.go [hunk 1]
```diff
@@ -75,6 +77,9 @@ func (ati *activityTrackerImpl) Stats() Activities {
 			}{Info: a, Count: count})
 		}
 	}
+	sort.Slice(activities, func(i, j int) bool {
+		return strings.Compare(strings.ToLower(activities[i].Info.ActivityType), strings.ToLower(activities[j].Info.ActivityType)) < 0
```
Comment (Groxx) [inline internal/common/debug/example_test.go:81]: Sorting makes perfect sense, but there's no reason to `ToLower` here afaict.  It'll just make things more ambiguous than they are IRL, where strings need to be identical in both workers and example-test output checks.

Comment (ketsiambaku) [reply internal/common/debug/example_test.go]: changed

**Review by Groxx — APPROVED** [review]
From the docs on `strings.Compare` it looks like we can/probably should just use `<` here... but they appear to be identical so I'll stamp now and land whichever you prefer :)  thankfully it's not a locale-sensitive comparison at least.

**Comment by ketsiambaku** [comment]
> From the docs on `strings.Compare` it looks like we can/probably should just use `<` here... but they appear to be semantically identical so I'll stamp now and land whichever you prefer :) thankfully it's not a locale-sensitive comparison at least.

I saw the last comment after merging. Followed up in #1367

**Pull request merged**